### PR TITLE
fix(): Fix categories form

### PR DESCRIPTION
**Pull request by Pralish**
https://github.com/restarone/violet_rails/issues/57

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -29,5 +29,8 @@ class Application < Rails::Application
 
     # since web console usage depends on subdomain rules
     config.web_console.development_only = false
+
+    # generates remote forms with form_with 
+    config.action_view.form_with_generates_remote_forms = true
```
Comment (donrestarone) [inline config/application.rb:34]: @Pralish you mentioned that the fix was to:

    Either add 'local: false' to the form
    OR update the config [Implemented]

Can we only add the local flag? I feel like setting the config could block us from Rails upgrades down the road

Comment (Pralish) [reply config/application.rb:34]: @donrestarone , We will have to add local flag to comfertable_mexican_sofa repo. Is that ok?

Comment (donrestarone) [reply config/application.rb:34]: @Pralish yeah lets do that and add a test for the redirect!

Comment (donrestarone) [reply config/application.rb:34]: no hurry, take your time on the change since we have a queue to drain @Pralish

**Pull request closed without merging**